### PR TITLE
Connect auth landing page to event dashboard and fix some styling

### PR DESCRIPTION
**Pull request by JasonYG**
[Notion task](https://www.notion.so/uwblueprintexecs/9224caeb21ce4350b8a1dc378d393323?v=43f44ac04d734394b7f7edc636c523b4)

## Changes
- Fixed styling for the auth landing page
- Clicking on an event card now redirects to the event dashboard

## Testing
- Click on an event card, and verify that it's redirecting to the correct URL

Note: the event dashboard isn't in this PR yet, so clicking on an event card won't actually change your screen
Note2: there's a small styling bug in this PR. The Add New Event button can cover an event card if there's enough of them. This should be fixed in a future PR.

## Screenshots
Landing Page
image

### CODE REVIEW

File: src/components/HomeLandingPage/HomeLandingPage.tsx [hunk 1]
```diff
@@ -28,6 +30,7 @@ const HomeLandingPage = () => {
   // Fetch events from cache
   const { data } = useQuery(GET_ALL_EVENTS);
   const events: Array<EventType> = data ? data.events : [];
+  console.log(events);
```
Comment (petorsong) [inline src/components/HomeLandingPage/HomeLandingPage.tsx]: can you check if the linter catches this? i thought i had this rule on

**Review by petorsong — CHANGES_REQUESTED** [review]
- if the `useeffect` error has a really easy fix, i wouldn't mind it if it was thrown into this PR too
- the selected tab still has the old dark/black colour - is this also supposed to match the blue?

File: src/components/HomeLandingPage/CardOptions.tsx [hunk 2]
```diff
@@ -19,6 +19,7 @@ const CardOptions: CardOptions = () => {
   const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
 
   const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    event.stopPropagation();
```
Comment (wongcmichelle) [inline src/components/HomeLandingPage/CardOptions.tsx:22]: What does adding this do?

File: src/components/HomeLandingPage/HomeLandingPage.tsx [hunk 3]
```diff
@@ -4,12 +4,13 @@ import Typography from '@material-ui/core/Typography';
...
-import useAllEvents from '../graphql/queries/hooks/events';
-import { EventType, GET_ALL_EVENTS } from '../graphql/queries/events';
-import '../styles/HomeLandingPage.css';
+import MenuTabs from '../common/MenuTabs';
+import AddEventButton from './AddEventButton';
+import EventCard from './EventCard';
+import useAllEvents from '../../graphql/queries/hooks/events';
+import { EventType, GET_ALL_EVENTS } from '../../graphql/queries/events';
+import '../../styles/HomeLandingPage.css';
+import { useHistory } from 'react-router-dom';
```
Comment (wongcmichelle) [inline src/components/HomeLandingPage/HomeLandingPage.tsx]: I'm getting a linter error for the order of the imports. `react-router-dom` import should occur before import of `../common/MenuTabs`

File: src/components/HomeLandingPage/HomeLandingPage.tsx (hunk 1 above)
Comment (wongcmichelle) [reply src/components/HomeLandingPage/HomeLandingPage.tsx]: I'm getting the linter error for this console.log

**Review by wongcmichelle — CHANGES_REQUESTED** [review]
It's working functionality wise :) Perhaps go through these files and lint them?

Also I'm assuming we're not fixing all styling issues here, but I'll point out two quick changes that would match the design better
- `indicatorColor` in `MenuTabs.tsx` should be "secondary"
- Missing some margin/bottom that prevents the "Add New Event" button from covering the event cards in the last row

File: src/components/HomeLandingPage/CardOptions.tsx (hunk 2 above)
Comment (petorsong) [reply src/components/HomeLandingPage/CardOptions.tsx:22]: i think this makes it so clicking the options won't also trigger the card onclick (which would take you to the event dashboard) - if so we should add a comment in code just to make it clear

File: src/components/HomeLandingPage/HomeLandingPage.tsx (hunk 1 above)
Comment (JasonYG) [reply src/components/HomeLandingPage/HomeLandingPage.tsx]: my bad, I forgot to run the linter

File: src/components/HomeLandingPage/CardOptions.tsx (hunk 2 above)
Comment (JasonYG) [reply src/components/HomeLandingPage/CardOptions.tsx:22]: yeah Peter's right, adding this prevents the user from being redirected to the event dashboard when they click on options

**Comment by JasonYG** [comment]
> It's working functionality wise :) Perhaps go through these files and lint them?
> 
> Also I'm assuming we're not fixing all styling issues here, but I'll point out two quick changes that would match the design better
> 
> * `indicatorColor` in `MenuTabs.tsx` should be "secondary"
> * Missing some margin/bottom that prevents the "Add New Event" button from covering the event cards in the last row

could you post a screenshot of the Add New Event issue? I'm not seeing the problem on my end

**Review by petorsong — DISMISSED** [review]

**Comment by wongcmichelle** [comment]
> > It's working functionality wise :) Perhaps go through these files and lint them?
> > Also I'm assuming we're not fixing all styling issues here, but I'll point out two quick changes that would match the design better
> > 
> > * `indicatorColor` in `MenuTabs.tsx` should be "secondary"
> > * Missing some margin/bottom that prevents the "Add New Event" button from covering the event cards in the last row
> 
> could you post a screenshot of the Add New Event issue? I'm not seeing the problem on my end

If there were one additional event card, it would be covered by the bottom. One additional styling issue. The `Tabs` component should have the prop `textColor="secondary"`.

image

File: src/components/common/MenuTabs.tsx [hunk 4]
```diff
@@ -41,7 +40,7 @@ const MenuTabs: React.FC<{
   return (
     <Tabs
       value={currentTab}
-      indicatorColor="primary"
+      indicatorColor="secondary"
       textColor="inherit"
```
Comment (wongcmichelle) [inline src/components/common/MenuTabs.tsx]:
```suggestion
      textColor="secondary"
```

File: src/components/HomeLandingPage/HomeLandingPage.tsx [hunk 5]
```diff
@@ -54,12 +56,13 @@ const HomeLandingPage = () => {
       <div className="landing-body">
```
Comment (wongcmichelle) [inline src/components/HomeLandingPage/HomeLandingPage.tsx:56]: The css for this "landing-body" class needs `padding: 70px 56px 168px 56px;`

**Review by petorsong — APPROVED** [review]

**Comment by petorsong** [comment]
>Note2: there's a small styling bug in this PR. The Add New Event button can cover an event card if there's enough of them. This should be fixed in a future PR.

we're gonna put this as a separate bug and deal with it in a separate ticket, since it may involve design input

**Review by wongcmichelle — APPROVED** [review]
LGTM

**Pull request merged**